app/src/main/java/com/framgia/englishconversation/screen/videoDetail/VideoDetailViewModel.java

```diff
@@ -56,13 +58,20 @@ public void onItemUserNameClick(TimelineModel item) {
                             .equals(item.getCreatedUser().getId())) {
                         return;
                     }
-                    mContext.startActivity(
-                            ProfileUserActivity.getInstance(mContext, item.getCreatedUser()));
+                    mContext.startActivity(ProfileUserActivity.getInstance(
```

Comment:
ERROR: 'method def' child have incorrect indentation level 20, expected level should be one of the following: 12, 16.